packages/opentelemetry-test-utils/package.json

```diff
@@ -33,7 +33,7 @@
     "typescript": "4.3.2"
   },
   "dependencies": {
-    "@opentelemetry/core": "^0.21.0",
-    "@opentelemetry/tracing": "^0.21.0"
+    "@opentelemetry/core": "0.21.0",
+    "@opentelemetry/tracing": "0.21.0"
```

Comment:
No idea why renovate wants to pin otel dependencies here (in all other modules it pins only dev dependencies).
Maybe because this module is only used internal and never published so in the end it's something like a dev-dependency.